packages/i18n/README.md

```diff
@@ -1,8 +1,8 @@
 # Internationalization (i18n)
 
-Internationalization utilities for client-side localization.
+Internationalization utilities for client-side localization. See [How to Internationalize Your Plugin](https://developer.wordpress.org/plugins/internationalization/how-to-internationalize-your-plugin/) for server-side documentation.
 
-https://codex.wordpress.org/I18n_for_WordPress_Developers
+For a complete example, see the [Internationalization section of the Gutenberg Handbook](https://wordpress.org/gutenberg/handbook/designers-developers/developers/internationalization.md).
```

Comment:
Can we just point to this URL only instead of duplicating documentation here?

Also, it's just `https://wordpress.org/gutenberg/handbook/designers-developers/developers/internationalization` without `.md` at the end.

Reply:
I removed the duplicate documentation and added a link back. 👍